src/app/clusters/group-key-mgmt-server/tests/TestGroupKeyManagementCluster.cpp

```diff
@@ -95,7 +100,174 @@ TEST_F(TestGroupKeyManagementCluster, AttributesTest)
 
     // There are only mandatory attributes in this cluster, so it should match the ones in Metadata exactly
     ASSERT_EQ(expectedBuilder.AppendElements(GroupKeyManagement::Attributes::kMandatoryMetadata), CHIP_NO_ERROR);
-    ASSERT_TRUE(Testing::EqualAttributeSets(attributesBuilder.TakeBuffer(), expectedBuilder.TakeBuffer()));
+    ASSERT_TRUE(chip::Testing::EqualAttributeSets(attributesBuilder.TakeBuffer(), expectedBuilder.TakeBuffer()));
```

Comment:
We should start using bits from https://github.com/project-chip/connectedhomeip/blob/master/src/app/clusters/testing/ValidateGlobalAttributes.h#L84

In particular IsAttributeListEqualTo will auto-handle mandatory elements, so that will make the code easier to read.

See example usage in https://github.com/project-chip/connectedhomeip/blob/d886e3a8a148106b88d685eec8c04983b58bbbbd/src/app/clusters/basic-information/tests/TestBasicInformationCluster.cpp#L84